### PR TITLE
[NGPDC-2597] confirm PU point component

### PR DESCRIPTION
**Pull request by pieliedie**
# Merge request description template

## Description

**1) Root cause**

**2) Changes**

Add new Confirm PU Point component.

image
image

**3) Impact**

## Check list

- [x] 1. Did you start and verify your changes?
- [x] 2. Did you run build to make sure that your changes can be built successfully?
- [x] 3. No !important flag, inline style in css
- [x] 4. No boilerplate codes (1)
- [x] 5. No duplicate code that exist in common functions?
- [x] 6. All of defined variables should have default value, check null and undefined?
- [x] 7. Use meaningful name for variables, no suffix 1,2,... Describes reference information for easier to understand what's inside. (2)
- [x] 9. Unsubscribed all of subscribers and even listeners when you don't need them.

```
(1)
Boilerplate codes is duplicated multiple times a bunch of the same code. This should be a common function to reuse through your code or you can use generic class / methods or design pattern to avoid the Boilerplate codes.
```

```
(2)
Follow theo coding convention
NO acronym variable name:
WRONG: el, elm
RIGHT: element
Your code can be read as a sentence
```

### CODE REVIEW

File: packages/react-compass/src/confirm-pickup/confirm-pickup.tsx [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+                fill='#0142AF'
+              />
+            </svg>
+
+            {specificPUPoint}
+          </div>
+        ) : (
+          <TextField
+            placeholder='E.g. Lobby, side entrance, etc'
```
Comment (hoapn-fsoft) [inline packages/react-compass/src/confirm-pickup/confirm-pickup.tsx:106]: placeholder should be taken from prop

File: packages/react-compass/src/confirm-pickup/confirm-pickup.tsx [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+            onChange={handleEditPUPoint}
```
Comment (hoapn-fsoft) [inline packages/react-compass/src/confirm-pickup/confirm-pickup.tsx:107]:
```suggestion
            onChange={handleChangeInput}

```

**Review by hoapn-fsoft — CHANGES_REQUESTED** [review]

File: packages/react-compass/src/confirm-pickup/confirm-pickup.tsx (hunk 1 above)
Comment (pieliedie) [reply packages/react-compass/src/confirm-pickup/confirm-pickup.tsx]: fixed. Thanks @hoapn-fsoft

**Review by hoapn-fsoft — APPROVED** [review]
LGTM

**Pull request merged**